x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/get_simple_rule_output_with_web_hook_action.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { NOTIFICATION_DEFAULT_FREQUENCY } from '@kbn/security-solution-plugin/common/constants';
+import { getSimpleRuleOutput } from './get_simple_rule_output';
+import { RuleWithoutServerGeneratedProperties } from './remove_server_generated_properties';
+
+export const getSimpleRuleOutputWithWebHookAction = (
```

Comment:
Since these were only used in the add_action.ts and update_action.ts which seemed to have been part of the migration, can we remove this util from the old folder?

Reply:
Note: being addressed in follow up PR to unblock others.